mrmustard/lab_dev/transformations/base.py

```diff
@@ -521,7 +521,7 @@ def random(cls, modes: Sequence[int], max_r: float = 1.0) -> Channel:
         U = Unitary.random(range(3 * m), max_r)
         u_psi = Vacuum(range(2 * m)) >> U
         A = u_psi.ansatz
-        kraus = A.conj[range(2 * m)] @ A[range(2 * m)]
+        kraus = A.conj.contract(A, range(2 * m), range(2 * m))
```

Comment:
can we rename A to ans? otherwise it can be confused with the A matrix

Reply:
Done in this PR: https://github.com/XanaduAI/MrMustard/pull/532